src/asset-swapper/utils/market_operation_utils/constants.ts

```diff
@@ -530,6 +530,8 @@ export const MAINNET_TOKENS = {
     agEUR: '0x1a7e4e63778b4f12a199c062f3efdd288afcbce8',
     ibEUR: '0x96e61422b6a9ba0e068b6c5add4ffabc6a4aae27',
     YFI: '0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e',
+    // Cronos (Crypto.com)
+    CRO: '0xa0b73e1ff0b80914ab6fe0444e65848c4c34450b',
```

Comment:
what do you mean?

Reply:
this constant is mainly for things like Curve pool where we need to specify tokens available pool.
in the case of CRO (unless this constant is going to be used somewhere) this won't do anything and can be deleted.

Reply:
It will frequently be an intermediateToken on DefiSwap/CryptoCom. Should we address that?

Reply:
In order to use CRO as an intermediate token we would also need to update the token adjacency graph. 

We should check whether it would be worth adding CRO as an intermediate token since that would significantly increase the overall gas usage. 

I don't think Defiswap had a lot of liquidity so probably it isn't worth it.